gridscale/resource_gridscale_server.go

```diff
@@ -758,13 +756,24 @@ func resourceGridscaleServerUpdate(d *schema.ResourceData, meta interface{}) err
 		if err != nil {
 			return fmt.Errorf("%s error: %v", errorPrefix, err)
 		}
+
+		//Update relationship between the server and networks
+		err = serverDepClient.UpdateNetworksRel(ctxWTimeout)
```

Comment:
I see this function is used two times: one with `shutdown` and the other one without `shutdown`. Would you explain the difference?

Reply:
Nice find. I will check this dup.